File: C++/Processor.cpp
#include <string>

using namespace std;

class Processor
{
    private:
        //atribut
        string nama;
        int price;

    public:
        //constructor
        Processor(){ }

        Processor(string nama, int price){
            this->nama = nama;
            this->price = price;
        }

        //prosedur setter data
        void setNama(string nama){
            this->nama = nama;
        }

        void setPrice(int price){
            this->price = price;
        }

        //prosedur getter
        string getNama(){
            return this->nama;
        }

        int getPrice(){
            return this->price;
        }
        
        //destruktor
        ~Processor(){
        }

};

File: PHP/processor.php
<?php

    class processor
    {
        private $nama = "";
        private $price = 0;

        #konstruktor
        public function __construct($nama = "", $price = 0){
            $this->nama = $nama;
            $this->price = $price;
        }

        #Prosedur setter data
        public function setNama($nama){
            $this->nama = $nama;
        }

        public function setPrice($price){
            $this->price = $price;
        }

        #prosedur getter data
        public function getNama(){
            return $this->nama;
        }

        public function getPrice(){
            return $this->price;
        }

        public function __destruct(){ }

    }

?>